### DISCUSSION TITLE
Welcome to Elrond-stable-diffusion-webui-discord Discussions!

### DISCUSSION THREAD

**Discussion by JumpingCoconut** [JumpingCoconut/Elrond-stable-diffusion-webui-discord#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AndreySkyFoxSidorov** [comment]
Hi. I would like to express my gratitude for your work. BOT is COOL! 
I think you are doing a very cool job and I want to thank you for what you are doing.
I would like to offer some ideas to improve your BOT. It would be cool to add options like: 
1. cfg_scale
2. steps
3. batch_size
4. Sampler

I think this will help improve the functionality of your work and make it even more useful for users.

T.H.X.

**Reply by JumpingCoconut** [reply]
Hello! Thank you for the kind words and the feedback!

It should take these values from your default config in automatic1111 webui. Check the file ``ui-config.json`` in your webui folder.

I didnt want the users to have to configure everything twice so I took just the default config. If this doesnt work let me know...
